docs/source/modules/socrates.rst

```diff
@@ -0,0 +1,215 @@
+| ``dt_rad``                 | 0        | Radiation timestep - every step if ``dt_rad<dt_atmos``                                                  |
++----------------------------+----------+---------------------------------------------------------------------------------------------------------+
+| ``store_intermediate_rad`` | True     | Keep rad constant over entire dt_rad? (recommended)                                                     |
++----------------------------+----------+---------------------------------------------------------------------------------------------------------+
+| ``dt_rad_avg``             | -1       | If averaging, over what time? ``dt_rad_avg=dt_rad`` if ``dt_rad_avg<=0``                                |
++----------------------------+----------+---------------------------------------------------------------------------------------------------------+
+| ``chunk_size``             | 16       | Number of gridpoints to pass to Socrates at a time                                                      |
+|                            |          | (see `README.md <https://github.com/ExeClim/Isca/blob/master/exp/test_cases/socrates_test/README.md>`_) |
++----------------------------+----------+---------------------------------------------------------------------------------------------------------+
+
+Spectral files
+^^^^^^^^^^^^^^
+
+Socrates reads external input spectral files that tell it the number of spectral bands to use, with one file setting the shortwave options, and another file setting the longwave options. Some spectral files have lots of bands, which will make the model run slowly. The default files used in the Met Office's Unified Model-GA7, and also in Isca, can be found within the ``data/spectra`` directory of Socrates source code. For example, it can be here if you put the Socrates source code within the ``trunk`` directory:
+::
+  src/atmos_param/socrates/src/trunk/data/spectra/ga7/sp_lw_ga7 for the longwave
+  src/atmos_param/socrates/src/trunk/data/spectra/ga7/sp_sw_ga7 for the shortwave
+
+or here if you have set ``GFDL_SOC`` as an environment variable:
+::
+  $GFDL_SOC/data/spectra/ga7/sp_lw_ga7 for the longwave
+  $GFDL_SOC/data/spectra/ga7/sp_sw_ga7 for the shortwave
+
++--------------------------------+---------------+-----------------------------------------------------+
+| Name                           | Default       | Description                                         |
++================================+===============+=====================================================+
+| ``socrates_hires_mode``        | False         | If false then run in 'GCM mode', and                |
+|                                |               | if true then usehigh-res spectral file              |
++--------------------------------+---------------+-----------------------------------------------------+
+| ``lw_hires_spectral_filename`` | 'unset'       |                                                     |
```

Comment:
Hi Qun - one final comment from me. Do you think you could add a description here for the two hires files? Something like:
```Longwave spectral file if `socrates_hires_mode` is True.``` and the same for the sw one. You could maybe move the hires ones below the normal ones in this table, so that a short comment like this makes more sense.

Reply:
Good suggestion. I have updated the table.